Dockerfile

```diff
@@ -4,7 +4,7 @@
 # [START cloudrun_rails_base_image]
 # Pinning the OS to buster because the nodejs install script is buster-specific.
 # Be sure to update the nodejs install command if the base image OS is updated.
-FROM ruby:3.0-buster as base
+FROM ruby:3.2.1-buster as base
```

Comment:
Can you change this to: `FROM ruby:3.2.1 as base`
since the 'buster' version gives an error for me